service/history/queues/executable.go

```diff
@@ -398,9 +406,17 @@ func (e *executableImpl) HandleErr(err error) (retErr error) {
 		return nil
 	}
 
+	e.attemptsWithUnexpectedError++
+	if e.attemptsWithUnexpectedError >= e.attemptsBeforeSendingToDlq() && e.dlqEnabled() {
+		e.logger.Error("Marking task as terminally failed after maximum number of attempts with unexpected errors, will send to DLQ",
```

Comment:
Can you extract a constant for the log message used above "Marking task as terminally failed, will send to DLQ" and reuse it here, and then add a comment that this message should be kept in sync with the one in the operational guide in `develop/docs/dlq.md`? If this is different, people might miss it because we tell users to search for that exact string when looking for DLQ reasons.